noria-server/dataflow/src/ops/mod.rs

```diff
@@ -21,6 +21,7 @@ pub enum NodeOperator {
     Sum(grouped::GroupedOperator<grouped::aggregate::Aggregator>),
     Extremum(grouped::GroupedOperator<grouped::extremum::ExtremumOperator>),
     Concat(grouped::GroupedOperator<grouped::concat::GroupConcat>),
+    FilterSum(grouped::GroupedOperator<grouped::filteraggregate::FilterAggregator>),
```

Comment:
It's a little weird for this to be called `FilterSum` when the inner type is `FilterAggregator` and supports both sum and count. Could this just be `Filtered` maybe?

Reply:
Yeah, I agree with that. I don't fully understand what this file is doing. I was just trying to be consistent with the above, where it defines Sum on Aggregator even though that also supports sum and count. Can you shed any light on why it's doing that?

Reply:
Ah, so, the type here is just to have a single type that can contain all operators. That's necessary when we, for example, need a type for the data-flow graph as a hole.

Reply:
OK. I'm still not understanding why one of its enum options is Sum instead of Aggregation?